pkg/helm/helm.go

```diff
@@ -78,13 +82,23 @@ func (h *Helm) FindVersions() error {
 	return err
 }
 
-// getReleasesVersionTwo retrieves helm 2 releases from ConfigMaps
+// getReleasesVersionTwo retrieves helm 2 releases from ConfigMaps or Secrets
 func (h *Helm) getReleasesVersionTwo() error {
+	var helmClient *helmstoragev2.Storage
 	if h.Version != "2" {
 		return fmt.Errorf("helm 2 function called without helm 2 version set")
 	}
-	hcm := driverv2.NewConfigMaps(h.Kube.Client.CoreV1().ConfigMaps(""))
-	helmClient := helmstoragev2.Init(hcm)
+	// helm store it's data in configmaps by default, but can be configured
+	// to use secrets instead (v2 only)
+	if h.Store == "secrets" {
+		opts := metav1.ListOptions{LabelSelector: "OWNER=TILLER"}
+		h.Kube.Client.CoreV1().Secrets(h.Namespace).List(opts)
+		hs := driverv2.NewSecrets(h.Kube.Client.CoreV1().Secrets(h.Namespace))
+		helmClient = helmstoragev2.Init(hs)
+	 } else {
```

Comment:
since the setting can be either `configmaps` or `secrets`, I think a switch statement would be better than if/else, so we can catch the possible error of `h.Store` being set to something unexpected.